runtime/lua/vim/lsp/util.lua

```diff
@@ -1620,18 +1620,14 @@ function M.open_floating_preview(contents, syntax, opts)
     api.nvim_buf_set_var(bufnr, 'lsp_floating_preview', floating_winnr)
   end
 
-  local augroup_name = ('closing_floating_preview_%d'):format(floating_winnr)
-  local ok =
-    pcall(api.nvim_get_autocmds, { group = augroup_name, pattern = tostring(floating_winnr) })
-  if not ok then
+  local augroup_name = 'nvim_closing_floating_preview'
+  if vim.fn.exists(('#%s'):format(augroup_name)) == 0 then
     api.nvim_create_autocmd('WinClosed', {
       group = api.nvim_create_augroup(augroup_name, {}),
```

Comment:
We could use `clear=true` here, which avoids all the `if exists(...)` code. But doesn't need to block this PR.

Reply:
if i remember correctly default is clear =true